tests/src/test/java/alluxio/stress/StressWorkerBenchIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress;
+
+import static org.junit.Assert.assertTrue;
+
+import alluxio.stress.cli.report.GenerateReport;
+import alluxio.stress.cli.worker.StressWorkerBench;
+import alluxio.testutils.BaseIntegrationTest;
+import alluxio.testutils.LocalAlluxioClusterResource;
+
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.TestRule;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Tests {@link StressWorkerBench}.
+ */
+public class StressWorkerBenchIntegrationTest extends BaseIntegrationTest {
+  @ClassRule
+  public static LocalAlluxioClusterResource sLocalAlluxioClusterResource =
+      new LocalAlluxioClusterResource.Builder().build();
+
+  @Rule
+  public TestRule mResetRule = sLocalAlluxioClusterResource.getResetResource();
+
+  @Rule
+  public TemporaryFolder mFolder = new TemporaryFolder();
+
+  @Test
+  public void readArray() throws Exception {
+    // Only in-process will work for unit testing.
+    String output = new StressWorkerBench().run(new String[] {
+        "--in-process",
+        "--start-ms", Long.toString(System.currentTimeMillis() + 1000),
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/stress-worker-base/",
+        "--threads", "2",
+        "--file-size", "1m",
+        "--block-size", "128k",
+        "--warmup", "0s", "--duration", "1s",
+    });
+
+    generateAndVerifyReport(Collections.singletonList("Worker Throughput"), output);
+  }
+
+  private void generateAndVerifyReport(List<String> expectedGraphNames, String... outputJson)
```

Comment:
generateAndVerifyReport looks exactly the same between all 3 of these integration tests and probably would benefit from having a common super class.

Reply:
Thanks! I forgot to refactor this out... I fixed it now.